doc/minting-distribution-tests.csv

```diff
@@ -5008,3 +5008,508 @@ Run #5
 999,2
 1000,2
 
+Run #5
```

Comment:
Lol don't commit this shit :)

I'll change the test it doesn't make any sense the way it is, I think I'll do it as a hardhat task and modify the `full-mint` test to do the fullmint without writing data.

Reply:
sounds good. deleted the file.

Reply:
The file could've stayed, but you committed bad data in it. 
Doesn't matter anyway.